examples/game/lib/game_demo_node.dart

```diff
@@ -434,20 +439,41 @@ class MovingEnemy extends Obstacle {
     constraints = [new ConstraintRotationToMovement(0.0, 0.5)];
   }
 
+  final double _swirlSpacing = 80.0;
+
+  _addRandomSquare(List<Offset> offsets, double x, double y) {
+    double xMove = (randomDouble() < 0.5) ? _swirlSpacing : -_swirlSpacing;
+    double yMove = (randomDouble() < 0.5) ? _swirlSpacing : -_swirlSpacing;
+
+    if (randomDouble() < 0.5) {
+      offsets.add(new Offset(x, y));
```

Comment:
offsets.addAll([
   new Offset(...),
   new Offset(...),
   new Offset(...),
]);

works too and might be slightly cleaner.